docs/index.rst

```diff
@@ -42,4 +32,7 @@ Robel Geda (`@robelgeda <https://github.com/robelgeda>`_).
 
```

Comment:
I think a section header above the table of contents would be helpful. Please paste something like the following...
```
Using STIPS
-----------
```
...above the `.. toctree::` line in this file.

Reply:
Done.